packages/server/src/runHttpQuery.ts

```diff
@@ -260,9 +260,11 @@ export async function runHttpQuery<TContext extends BaseContext>({
       ...graphQLResponse.http,
       body: {
         kind: 'complete',
-        string: prettyJSONStringify(
-          orderExecutionResultFields(graphQLResponse.body.singleResult),
-        ),
+        string: internals.stringifyResult
+          ? internals.stringifyResult(graphQLResponse.body.singleResult)
+          : prettyJSONStringify(
+              orderExecutionResultFields(graphQLResponse.body.singleResult),
+            ),
```

Comment:
Sorry I didn't catch this earlier. I'd like to preserve the `orderExecutionResultFields` call in both cases since that's a spec concern. So maybe we do this instead, let me know what you thing:
* make the `internals.stringifyResult` not optional (keep the configuration optional)
* default the internals property to `prettyJSONStringify` if it's not configured
* change the call here to just:
```ts
string: internals.stringifyResult(
  orderExecutionResultFields(
    graphQLResponse.body.singleResult,
  ),
)
```

Reply:
Makes sense to me -- making change now

Reply:
Take a peak? :)